<ui version="4.0" >
<comment> Begin CVS Header </comment>
<comment>   $Source: /fs/turing/cvs/copasi_dev/copasi/wizard/wizard.ui,v $ </comment>
<comment>   $Revision: 1.8 $ </comment>
<comment>   $Name: Build-33 $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2010/07/16 19:06:34 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2010 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>WizardDialog</class>
 <widget class="QDialog" name="WizardDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Tutorial</string>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QPushButton" name="button1" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text" >
      <string>Step 1
Naming and Description</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
     <property name="autoDefault" >
      <bool>false</bool>
     </property>
     <property name="default" >
      <bool>false</bool>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="buttonGroupId" stdset="0" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item rowspan="7" row="0" column="1" >
    <widget class="QTextBrowser" name="textBrowser" />
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="button2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Step 2
Creating Reactions</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="autoDefault" >
      <bool>false</bool>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="buttonGroupId" stdset="0" >
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="button3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Step 3
Refining the Model</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="autoDefault" >
      <bool>false</bool>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="buttonGroupId" stdset="0" >
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QPushButton" name="button4" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Step 4
Calculating a Time Course</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="autoDefault" >
      <bool>false</bool>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="buttonGroupId" stdset="0" >
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QPushButton" name="button5" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Step 5
Defining the Output</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="autoDefault" >
      <bool>false</bool>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="buttonGroupId" stdset="0" >
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QPushButton" name="button6" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Step 6
Plotting a Trajectory</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="autoDefault" >
      <bool>false</bool>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="buttonGroupId" stdset="0" >
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="backButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&lt;&lt; back</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="forwardButton" >
       <property name="text" >
        <string>forward >></string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>310</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>forwardButton</sender>
   <signal>clicked()</signal>
   <receiver>WizardDialog</receiver>
   <slot>forwardButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>WizardDialog</receiver>
   <slot>backButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>WizardDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
